### PR TITLE
Fixed enb make

### LINKED ISSUE #1190: enb make is broken
If run `enb make` then will get an error:

```
Error: There is no tech for target desktop.examples/page/10-simple/10-simple.bemjson.js.
    at Error (native)
    at TargetNotFoundError (/Users/blond/projects/bem-core/node_modules/enb/lib/errors/target-not-found-error.js:11:23)
    at /Users/blond/projects/bem-core/node_modules/enb/lib/node.js:537:27
    at Array.map (native)
    at /Users/blond/projects/bem-core/node_modules/enb/lib/node.js:533:36
    at Array.<anonymous> (/Users/blond/projects/bem-core/node_modules/enb/node_modules/vow/lib/vow.js:202:56)
    at Immediate.callFns [as _onImmediate] (/Users/blond/projects/bem-core/node_modules/enb/node_modules/vow/lib/vow.js:462:35)
    at processImmediate [as _immediateCallback] (timers.js:367:17)
```

### PR DESCRIPTION
**Pull request by blond**
Resolved #1190

### CODE REVIEW

**Comment by blond** [comment]
:up:

File: .enb/helpers/examples.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    }
+    var magic = project.module('enb-magic-factory'),
+        plugin = project.module('enb-bem-examples'),
+        // create task with `examples` name
+        // and get helper to configure it
+        configurator = plugin.createConfigurator(taskName),
+        helper = magic.getHelper(taskName);
+
+    PLATFORMS.forEach(function (platform) {
+        var dirPattern = platform + destExt + '/*/*';
```
Comment (tadatuta) [inline .enb/helpers/examples.js:37]: shouldn't it be `path.join(platform + destExt, '*', '*')`?

Comment (blond) [inline .enb/helpers/examples.js:37]: No, paths to node or targets always with `/`.

**Pull request merged**